src/olympia/addons/migrations/0020_auto_20200909_1302.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 2.2.16 on 2020-09-09 13:02
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("addons", "0019_auto_20200901_1459"),
+    ]
+
+    operations = [
+        migrations.AlterField(
```

Comment:
Interesting. So removing a field from a model doesn't cause a migration that deletes the field from the db, but rather just makes it nullable?

Reply:
No, I had to generate the migration for that before I removed the field in the class. When deleting a field in the class, it creates a migration that performs the column deletion in the DB but I don't want it here. It will be done in a different push (see #15446)